migrations/sql/V1.35__add_user_table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE minespace_users (
```

Comment:
This may be picky, but we typically don't pluralize table names. Suggest we call this table minespace_user. However, if it is a ton of work to refactor, we can leave it.

Reply:
changed,